elvis.config

```diff
@@ -22,7 +22,11 @@
                       sumo_find_SUITE,
                       sumo_basic_SUITE,
                       sumo,
-                      sumo_internal]}
+                      sumo_internal,
```

Comment:
You might want to remove the `invalid_dynamic_call` rule altogether, it's not a critical rule for this project and you're already ignore a lot of modules.